k8s/k8s_sars_cov_2.yaml

```diff
@@ -27,9 +27,10 @@ spec:
       - image: 566277102435.dkr.ecr.eu-west-2.amazonaws.com/congenica/psga-dev/sars-cov-2-pipeline:dev_latest
         name: sars-cov-2-pipeline
         command:
-        - python
+        - /bin/bash
         args:
-        - autoresumer.py
+        - -c
+        - "while true; do sleep 30; done;"
```

Comment:
@pdp10 is this intentional?

Reply:
Yes. Not that you guys do not use this file at all. 
Actually, this file will be removed at some point as the deployment is managed by BE and OPS.